README.md

```diff
@@ -20,18 +20,16 @@ Today, Kruise offers four workload controllers:
 
 - [UnitedDeployment](./docs/concepts/uniteddeployment/README.md): This controller manages application pods spread in multiple fault domains by using multiple workloads.
 
-Please see [documents](./docs/README.md) for more technical information.
-
 The project **roadmap** is actively updated in [here](https://github.com/openkruise/kruise/projects).
-
-Several [tutorials](./docs/tutorial/README.md) are provided to demonstrate how to use the workload controllers.
-A [video](https://www.youtube.com/watch?v=elB7reZ6eAQ) by [Lachlan Evenson](https://github.com/lachie83) is also provided for demonstrating the controllers.
+This [video](https://www.youtube.com/watch?v=elB7reZ6eAQ) demo by [Lachlan Evenson](https://github.com/lachie83) is great for new users.
 
 ## Getting started
 
 ### Check before installation
 
-You should check the cluster before installation via the command-line with either `curl` or `wget`.
+Kruise requires APIServer to enable features such as `MutatingAdmissionWebhook` admission webhook. You can check your cluster qualification
```

Comment:
`MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook`

Reply:
done